packages/adapter-node/src/server.js

```diff
@@ -40,6 +42,7 @@ export function createServer({ render }) {
 		: noop_handler;
 
 	const server = polka().use(
+		// @ts-ignore TODO - compression return doesn't play well with polka.use
```

Comment:
see https://github.com/lukeed/polka/issues/173. maybe the solution is to cast it to `unknown` and then to `polka.Middleware`?  either that or I guess someone would need to dig into to understand what is different about the Polka types